### PR TITLE
Chemistry Fun

### PR DESCRIPTION
**Pull request by Kurfursten**
- Slight dropper refactor; allows you to use it on any item you can use a syringe on.
- New NanomedPlus contraband: trail mix. 7 random pills from a list of 12. 4 good, 4 neutral, 4 bad.
- New NanomedPlus premium good: time release pills. These pills release their chemical payload after a time given by (three times sugar content) seconds.
- New Chemist traitor item: Hug Explosion Kit. Comes with a bottle of 30u Bicarodyne and syringe. Don't confuse it with Bicaridine, this stuff explodes when mixed with hugs. From a technical standpoint it was necessary to put in a box because the uplink seemed to have issues with spawning in the bottle directly.
- The Chief Medical Officer can now get Geneticist and Chemist traitor items.
- Brawndo can now be found in one of the xenoarcheology plants. It has what scientists crave!

### CODE REVIEW

File: code/modules/reagents/reagent_containers/glass/bottle.dm [hunk 1]
```diff
@@ -380,3 +380,13 @@
 	New()
 		..()
 		reagents.add_reagent("frostoil", 30)
+
+/obj/item/weapon/reagent_containers/glass/bottle/antisocial
+	//No special name or description
+	name = "bottle"
+	desc = "A small bottle."
+	icon = 'icons/obj/chemical.dmi'
+
```
Comment (Kurfursten) [inline code/modules/reagents/reagent_containers/glass/bottle.dm]: Before anyone else comments on it, yes, these are all three variables that get inherited from the parent as exactly that anyway. Yes, this is a waste of three lines. I'll look at it next commit.

**Comment by Shadowmech88** [comment]
While you're touching Chemist traitor items, maybe look at #5861?

**Comment by NigglyWiggly** [comment]
>this stuff explodes when mixed with hugs.

**Comment by Kurfursten** [comment]
@Shadowmech88 As the description implies when examined, these are special pills that can be modified with syringes and droppers.

File: code/modules/reagents/reagent_containers/pill.dm [hunk 2]
```diff
@@ -228,3 +231,51 @@
...
 		..()
 		reagents.add_reagent("hyperzine", 10)
+
+/obj/item/weapon/storage/pill_bottle/time_release
+	name = "controlled release pill bottle"
+	desc = "A bottle containing special pills which can be calibrated for delayed release with sugar."
+
+/obj/item/weapon/storage/pill_bottle/time_release/New()
+	..()
+	for(var/i=0,i<7,i++)
```
Comment (d3athrow) [inline code/modules/reagents/reagent_containers/pill.dm]: for future reference you can do ```for(var/i=0 to 7)```, quicker to write and tiny bit better performance. not a big deal though

File: code/modules/reagents/reagent_containers/pill.dm [hunk 3]
```diff
@@ -228,3 +231,51 @@
...
+
+/obj/item/weapon/reagent_containers/pill/time_release
+	name = "time release pill"
+	desc = "A pill which will not be metabolized until all of the sugar inside metabolizes. Unlike other pills, it is specially designed to be compatible with droppers and syringes."
+	icon_state = "pill18"
+
+/obj/item/weapon/reagent_containers/pill/time_release/injest(mob/M as mob)
+	var/timer = round(reagents.get_reagent_amount("sugar"),1)
+	forceMove(M)
+	spawn(timer*30)
```
Comment (d3athrow) [inline code/modules/reagents/reagent_containers/pill.dm]: This entire block needs sanity to prevent runtimes from the mob or reagents being not there.

**Comment by FudgePucker** [comment]
>exploding hugs

You are a god damn genius. :+1:

**Comment by Kurfursten** [comment]
@d3athrow Gotcha, changes made.

@FudgePucker That's the power of German engineering

**Comment by Kurfursten** [comment]
@d3athrow @N3X15 Barring any other objections, I'd like this PR to go through quickly so I can resolve the uplink_items conflicts it will create with my other PR (wheelchair-related).

File: code/modules/reagents/reagent_containers/pill.dm [hunk 4]
```diff
@@ -228,3 +233,51 @@
...
+	name = "time release pill"
+	desc = "A pill which will not be metabolized until all of the sugar inside metabolizes. Unlike other pills, it is specially designed to be compatible with droppers and syringes."
+	icon_state = "pill18"
+
+/obj/item/weapon/reagent_containers/pill/time_release/injest(mob/M as mob)
+	if(!reagents) return
+	if(!M) return
+	var/timer = round(reagents.get_reagent_amount("sugar"),1)
+	forceMove(M)
+	spawn(timer*30)
```
Comment (clusterfack) [inline code/modules/reagents/reagent_containers/pill.dm]: Are you aware that this is 3 seconds * timer? (Not everyone is aware of this fact)

Comment (Kurfursten) [inline code/modules/reagents/reagent_containers/pill.dm]: Yes, it says so in the changelog

Comment (clusterfack) [inline code/modules/reagents/reagent_containers/pill.dm]: Just making sure, thats normally why we use some defines like 3 SECONDS or 3 MINUTES

**Pull request merged**